BookingsApi/BookingsApi.Infrastructure.Services/Dtos/HearingDto.cs

```diff
@@ -14,5 +14,6 @@ public class HearingDto
         public string HearingVenueName { get; set;}
         public bool RecordAudio { get; set; }
         public string HearingType { get; set; }
+        public string JudgeName { get; set;}
```

Comment:
Why is the Judge name needed? Also below you use the DisplayName property, this should be named `JudgeDisplayName` to avoid confusion with their first or last name.

Also why not have a `Judge` property of type `ParticipantDto`. if this is needed only for the allocation event?

Reply:
the name is needed to pass to BS.
Changed the property name

Reply:
Then if it's only for the this integration event could you use a Judge property instead of changing the HearingDto to reflect something that isn't a hearing